docs/planner-usage.md

```diff
@@ -14,19 +17,62 @@ If you want to run any of the planners based on Fast Downward that
 participated in IPC 2011, please also check the page on
 [IPC planners](ipc-planners.md).
 
-## Caveats
+## Translate component
+To translate a PDDL input task into a SAS+ task without performing a search run the following:
+
+    ./fast-downward.py --translate [<path-to-domain>/domain.pddl] <path-to-problem>/problem.pddl
+
+Note:
+
+-   Giving the domain file as input is optional. 
+    In case no domain file is specified, the component will search for a file called `domain.pddl` located in `<path-to-problem>` directory. 
+-   The paths to the domain/task files can be absolute, relative or contain path variables.
+-   Translator component options can be specified after the input files following the `--translator-options` flag.
+
+## Search component
+To run a search on a PDDL input task run the following:
+
+    ./fast-downward.py [<path-to-domain>/domain.pddl] <path-to-problem>/problem.pddl --search "<some-search-algorithm>(<some-evaluator>)"
```

Comment:
there are search algos that expect no evaluator, but a list of evaluator or a list of search algorithms...
I'd suggest to just say `"<some-search-algorithm>()"`